### ISSUE TITLE
SISDR loss

### ISSUE THREAD

**Issue by fncode246** [csteinmetz1/auraloss#28]
Hi, Thanks for sharing this useful repo.
I want to use SISDR as loss function during training.
here is an example of target and input:
inputs = torch.rand(8,1,44100)
target = torch.rand(8,1,44100)
loss=SISDRLoss(inputs,target)
print(loss)

But it only returns  :  
SISDRLoss()
and I can't calculate SI SDR loss value with this.
Please help me...
Thanks

**Comment by csteinmetz1** [comment]
Hi, thanks for checking out auraloss. 

I think the issue is that you must first instantiate the SISDRLoss module first. 
For example:
```
inputs = torch.rand(8,1,44100)
target = torch.rand(8,1,44100)
sisdr_loss = SISDRLoss() # instantiate the loss module
loss = sisdr_loss(inputs,target) # measure the loss
print(loss)
``` 
Let me know if this addresses your issue.

**Comment by fncode246** [comment]
Yes, it works.
Thanks

**Comment by fncode246** [comment]
Is there any way to use SI-SDR loss with PIT (permutation invariant of sources during training source separation model)?

**Comment by csteinmetz1** [comment]
Sorry for the delay on responding to this. Currently we don't have support for PIT and likely won't add it. You could try using auraloss loss function with the PIT wrapper from 
Asteroid (https://github.com/asteroid-team/pytorch-pit/blob/master/torch_pit/pit_wrapper.py).
